<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_com">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>210</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>COM</string>
    </property>
    <widget class="QComboBox" name="comboBox_serial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>190</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>190</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud Rate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_baud">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_device">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>441</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QPushButton" name="pushButton_browse">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_filename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>390</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>190</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Firmware File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_probe">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Probe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_flash">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Flash</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_read">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_erase">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Erase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_jump">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Jump</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>199</y>
      <width>661</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>OUTPUT:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearoutput">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>170</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
